README.md

```diff
@@ -221,13 +221,13 @@ version and the whole actual API (paste into any [swagger file generator](https:
 ```
 curl http://localhost:3080/version
 
-{"version":"7.1.0"}
+{"version":"7.2.1"}
 ```
 
 ```
 curl http://localhost:3080/api-version
 
-{"api-version":"7.2.0"}
+{"api-version":"7.3.0"}
```

Comment:
we just forgot to count that up in 7.3.0? maybe its a good time to make at least this in sync with the released version

Reply:
Nah, let's keep it this way. If there is an update to _just_ the compiler we'd still have to bump the version of the HTTP-compiler but it would be wrong to bump the API version, right?

Reply:
maybe we should so http-compiler version == api-version and if there is a breaking api change also consider it a breaking http compiler change, so the http compiler is the api